docs/scos/dev/front-end-development/202307.0/oryx/building-pages/oryx-layout.md

```diff
@@ -0,0 +1,208 @@
+- transform: Specifies the transformation applied to the component.
+- font-size: Specifies the font size of the component.
+- font-weight: Specifies the font weight of the component.
+- line-height: Specifies the line height of the component.
+
+## Apply layouts
+
+There are different ways to leverage the Oryx layout system.
+
+- Configure layout in a page or composition
+- Configure layout in (some) components
+- Integrate the `<oryx-layout>` component in your code
+- Integrate the `LayoutMixin` in your code
+
+### Use layout in compositions
+
+The most common approach is to apply a layout to [compositions](/docs/scos/dev/front-end-development/{{page.version}}/oryx/building-pages/oryx-compositions.html). Compositions are integrated with the layout system, so that you can apply layout types and properties to the list of components that they host.
+
+[Pages](/docs/scos/dev/front-end-development/{{page.version}}/oryx/building-pages/oryx-pages.html) use compositions under the hood, which is why you can apply layout to the page options as well.
+
+The following example shows how a layout type, a layout option and layout styles are applied to a composition.
+
+```ts
+export const page = {
+  type: "Page",
+  components: [
+    {
+      type: "oryx-composition",
+      options: {
+        rules: [{ layout: "list", sticky: true }],
+      },
+    },
+  ],
+};
+```
+
+### Use layout component in your code
+
+You can use the layout component in your code and provide the layout and layout options:
+
+```html
+<oryx-layout layout="carousel" layout-sticky> ... </oryx-layout>
+```
+
+If you like to leverage component options, including layout options in the hardcode component, you can either provide an associated experience data `uid` or the `options` attribute.
+
+<!-- TODO: link to component options when this doc is available -->
+
+Another option would be to integrate with the `CompositionComponent` as the layout and options are automatically assigned based on the composition `uid`.
+
+```html
+<oryx-composition uid="component-uid"></oryx-layout>
+```
+
+This approach is used in some Oryx components, such as the `CartTotalsComponent`.
+
+### Use layout mixin in your code
+
+Another option to use layout in your components is by using the `LayoutMixin`. The `LayoutMixin` provides the layout component properties and integration the layout logic automatically, without adding the `oryx-layout` element.
+
+An example of using this approach can be found in the `ProductListComponent`. The product list component uses the layout mixin so that the layout can be applied to the component directly. The following code shows the integration of the `LayoutMixin`.
+
+```ts
+@hydrate()
+export class ProductListComponent extends LayoutMixin(LitElement) {
+  protected override render(): TemplateResult {
+    return html`
+      // write the component html here ${unsafeHTML(
+        `<style>${this.layoutStyles()}</style>`
+      )}
```

Comment:
maybe?
```suggestion
      // write the component html here 
      ${unsafeHTML(
        `<style>${this.layoutStyles()}</style>`
      )}
```

Reply:
prettier is in my way, but I'll do a small improvement to make it beter